.release/ci.hcl

```diff
@@ -0,0 +1,130 @@
+schema = "1"
+
+project "vault-plugin-secrets-azure" {
+  team = "vault"
+  slack {
+    // #vault-releases channel
+    notification_channel = "CRF6FFKEW"
+  }
+  github {
+    organization = "hashicorp"
+    repository = "vault-plugin-secrets-azure"
+    release_branches = [
+        "main"
+        "release/vault-1.8.x",
+        "release/vault-1.9.x",
+        "release/vault-1.10.x",
+        "release/vault-1.11.x",
+        "release/vault-1.12.x",
+        "vault-4378/crt", // for testing
+    ]
+
+  }
+}
+
+event "merge" {
+  // "entrypoint" to use if build is not run automatically
+  // i.e. send "merge" complete signal to orchestrator to trigger build
+}
+
+event "build" {
+  depends = ["merge"]
+  action "build" {
+    organization = "hashicorp"
+    repository = "vault-plugin-secrets-azure"
+    workflow = "build"
+  }
+}
+
+event "upload-dev" {
+  depends = ["build"]
+  action "upload-dev" {
+    organization = "hashicorp"
+    repository = "crt-workflows-common"
+    workflow = "upload-dev"
+    depends = ["build"]
+  }
+
+  notification {
+    on = "fail"
+  }
+}
+
+event "security-scan-binaries" {
+  depends = ["upload-dev"]
+  action "security-scan-binaries" {
+    organization = "hashicorp"
+    repository = "crt-workflows-common"
+    workflow = "security-scan-binaries"
+    config = "security-scan.hcl"
+  }
+
+  notification {
+    on = "fail"
+  }
+}
+
```

Comment:
Since we're producing darwin and windows binaries, do we need to notarize them like vault does? https://github.com/hashicorp/vault/blob/main/.release/ci.hcl#L90-L140